app/helpers/browse_helper.rb

```diff
@@ -3,8 +3,44 @@ def display_popular_links_for_slug?(slug)
     I18n.exists?(slug.to_s, scope: "browse.popular_links")
   end
 
+  def variant_a_popular_links(slug)
+    I18n.t(slug.to_s, scope: "browse.popular_links")
+  end
+
+  def variant_b_popular_links(slug)
+    I18n.t(slug.to_s, scope: "browse.popular_links")
+  end
+
+  def variant_c_popular_links(slug)
+    I18n.t(slug.to_s, scope: "browse.popular_links")
+  end
+
+  def variant_z_popular_links(slug)
+    I18n.t(slug.to_s, scope: "browse.popular_links")
+  end
+
+  def no_popular_tasks_ab_test?
+    !respond_to?(:popular_tasks_variant)
+  end
+
+  def variant_popular_links_for_slug(slug)
+    default_variant_popular_links = variant_c_popular_links(slug)
+
+    if no_popular_tasks_ab_test?
+      default_variant_popular_links
+    elsif popular_tasks_variant_a_page?
+      variant_a_popular_links(slug)
+    elsif popular_tasks_variant_b_page?
+      variant_b_popular_links(slug)
+    elsif popular_tasks_variant_z_page?
+      variant_z_popular_links(slug)
+    else
+      default_variant_popular_links
```

Comment:
The `default_variant_popular_links` is where you'll need to add some logic that figures out if the page should have the original links, or nothing at all. It could depend on the presence or absence of the `control` key in the yaml file or you could just hard code it (is the slug `benefits` or `business`?)

```
def default_variant_popular_links(slug)
 if original_page_had_popular_tasks?(slug)
   I18n.t("browse.popular_links.#{slug}.control") 
 else
   []
 end
end
```

Reply:
default_variant_popular_links has been removed.